apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/events/BaseRedisSubscriberTest.java

```diff
@@ -237,11 +240,86 @@ void shouldContinueProcessingAfterFailedMessages() {
             publishMessagesToStream(usualPayloadMessages);
 
             waitForMessagesProcessed(subscriber, usualPayloadMessages.size());
-            waitForMessagesAckedAndRemoved(otherPayloadMessages.size());
+            waitForMessagesAckedAndRemoved();
             assertThat(subscriber.getFailedMessageCount().get()).isEqualTo(otherPayloadMessages.size());
         }
     }
 
+    @Nested
+    class RetryTests {
+
+        @Test
+        void shouldAckAndRemoveAfterMaxRetries() {
+            var fastConfig = config.toBuilder()
+                    .claimIntervalRatio(2)
+                    .pendingMessageDuration(io.dropwizard.util.Duration.milliseconds(500))
```

Comment:
Replace fully qualified name with import:
```suggestion
                    .pendingMessageDuration(Duration.milliseconds(500))
```

Reply:
Not possible in this case, as two `Duration` classes are used: the Java SDK standard and the Dropwizard one. I can only import one, decided to keep the Java SDK one.